scripts/config/jest/index.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
```

Comment:
Same as above, this isn't a script, scripts should call out to the `src` directory for implementation stuff